src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -330,4 +335,30 @@ mod test {
             assert_eq!(got, exp);
         }
     }
+
+    #[test]
+    fn test_round() {
+        let tests = vec![
+            (ScalarFuncSig::RoundInt, Datum::I64(1), Datum::I64(1)),
```

Comment:
do we need to add some tests for more data types round? like round(1.2) -> 1

@AndreMouche

Reply:
No, we needn't do this in this PR since it just implements `RoundInt`. For round(1.2), it should be tested in `RoundDec` @siddontang